components/projectsView/ProjectFilters.js

```diff
@@ -76,16 +74,20 @@ export default function ProjectFilters({
     <Row className="text-center">
       {isTabletOrMobile ? (
         <Col key="projectStatusFilter">
-          <Row className={`bg-light h-100 `}>
+          <Row
+            className={`h-100`}
+            style={{
+              backgroundColor: "#f8f9fa",
+            }}
+          >
             <Col>
-              <Form.Control
+              <Form.Select
```

Comment:
Bootstrap got rid of the down arrow with Form.Control and prefer to use Form.Select, so the `as="select"` prop below can be removed as well